recipe/meta.yaml

```
@@ -80,7 +80,7 @@ requirements:
     - pytorch
     - pytorch * {{ proc_type }}*
     - libxc-c
-    - libxc-c * {{ proc_type }}_*
+    - libxc-c * cpu_*
 
   run:
     - {{ mpi }}
```

Think we may want this pinning in `run` too

```suggestion
    - {{ mpi }}
    - libxc-c * cpu_*
```